collectors/ebpf.plugin/ebpf_socket.c

```diff
@@ -289,30 +308,23 @@ static void ebpf_socket_send_data(ebpf_module_t *em)
 
     // We read bytes from function arguments, but bandiwdth is given in bits,
     // so we need to multiply by 8 to convert for the final value.
-    write_count_chart(
-      NETDATA_TCP_FUNCTION_COUNT, NETDATA_EBPF_FAMILY, socket_publish_aggregated, 3);
-    write_io_chart(
-        NETDATA_TCP_FUNCTION_BITS, NETDATA_EBPF_FAMILY, socket_id_names[0], common_tcp.write*8/1000,
-        socket_id_names[1], common_tcp.read*8/1000);
+    write_count_chart(NETDATA_TCP_FUNCTION_COUNT, NETDATA_EBPF_IP_FAMILY, socket_publish_aggregated, 3);
+    write_io_chart(NETDATA_TCP_FUNCTION_BITS, NETDATA_EBPF_IP_FAMILY, socket_id_names[0],
+                   common_tcp.write * 8/BITS_IN_A_KILOBIT, socket_id_names[1], common_tcp.read * 8/BITS_IN_A_KILOBIT);
```

Comment:
I suggest changing "sent" and "received" dimensions order:
- "received"
- "sent"

Check "Network Interfaces" charts

Reply:
I inverted the dimensions. :handshake: